.travis.yml

```diff
@@ -11,3 +14,13 @@ script:
   - paver install_pages
```

Comment:
This will be done by the call to `docker-compose build` below. If you build the image first, you can run all of the validation inside a container and avoid duplicating work. We do something similar for credentials: https://github.com/edx/credentials/blob/master/.travis.yml.